sktime/transformations/series/kalman_filter.py

```diff
@@ -533,7 +533,7 @@ class KalmanFilterTransformerPK(BaseKalmanFilter, BaseTransformer):
         "capability:missing_values:removes": False,
         # is transform result always guaranteed to contain no missing values?
         "scitype:instancewise": True,  # is this an instance-wise transform?
-        "python_dependencies": "pykalman",
+        "python_dependencies": "pykalman-bardo",
```

Comment:
```suggestion
        "python_dependencies": "pykalman-bardo",
        "python_dependencies_alias": {"pykalman-bardo": "pykalman"},
```

I think this is necessary.